src/Judopay/Model/TokenPayment.php

```diff
@@ -19,7 +19,8 @@ class TokenPayment extends \Judopay\Model
         'cv2' => DataType::TYPE_STRING,
         'consumerLocation' => DataType::TYPE_ARRAY,
         'mobileNumber' => DataType::TYPE_STRING,
-        'emailAddress' => DataType::TYPE_STRING
+        'emailAddress' => DataType::TYPE_STRING,
+        'clientDetails' => DataType::TYPE_ARRAY,
```

Comment:
So there's a bit of a problem here, the "clientDetails" property needs to serialize as a JSON object rather than a JSON array. Currently we haven't implemented DataType::TYPE_OBJECT but I'll see if we can sort that out.